mappings/net/minecraft/world/spawner/CatSpawner.mapping

```diff
@@ -0,0 +1,27 @@
+CLASS net/minecraft/class_4274 net/minecraft/world/spawner/CatSpawner
+	COMMENT A spawner for cats in villages and swamp huts.
+	COMMENT
+	COMMENT @implNote Cats in swamp huts are also spawned in
+	COMMENT {@link net.minecraft.world.gen.chunk.ChunkGeneratorSettings#getEntitySpawnList}.
+	FIELD field_19171 ticksUntilNextSpawn I
+	FIELD field_30601 SPAWN_COOLDOWN I
```

Comment:
Maybe like `INTERVAL` as a cooldown can be understood as the current ticks till next spawn.

Also `ticksUntilNextSpawn` is too verbose. I'd prefer `waitTicks` or `ticksLeft` instead (or `cooldown` if you rename the constant too)